### PR TITLE
Fix not executing multiple fork tasks

### PR DESCRIPTION
**Pull request by eisuke**
Fix the bug https://github.com/cookpad/kuroko2/issues/62

@cookpad/dev-infra please review.

### CODE REVIEW

File: lib/autoload/kuroko2/workflow/task/fork.rb [hunk 1]
```diff
@@ -32,6 +31,16 @@ def validate
 
         private
 
+        def children
+          token.context['fork_children_ids'] ||= {}
+          child_ids = token.context['fork_children_ids'][token.path] ||= []
+          if child_ids.present?
```
Comment (sorah) [inline lib/autoload/kuroko2/workflow/task/fork.rb]: using `empty?` is enough?

File: lib/autoload/kuroko2/workflow/task/fork.rb [hunk 2]
```diff
@@ -44,6 +53,7 @@ def create_child_token(child_node:, env: {})
           attributes['context']['ENV'] = (attributes['context']['ENV'] || {}).merge(env)
 
           Token.create!(attributes).tap do |created|
+            token.context['fork_children_ids'][token.path] << created.id
```
Comment (sorah) [inline lib/autoload/kuroko2/workflow/task/fork.rb]: this can be nil?

Comment (eisuke) [inline lib/autoload/kuroko2/workflow/task/fork.rb]: No, It is always Array.
But I came to feel it is hard to notice. fixed aad3d24

File: lib/autoload/kuroko2/workflow/task/fork.rb [hunk 3]
```diff
@@ -32,6 +31,19 @@ def validate
 
         private
 
+        def children
+          if fork_children_ids.present?
```
Comment (sorah) [inline lib/autoload/kuroko2/workflow/task/fork.rb]: this should be always Array; empty? is enough. `present?` doesn't perform a empty-ness check only.

Comment (eisuke) [inline lib/autoload/kuroko2/workflow/task/fork.rb]: Fixed it in 7d5f388 (Also fix redundant repeating `empty?`).

**Pull request merged**